nobl9/resource_slo.go

```diff
@@ -6,238 +6,342 @@ import (
+		"project":      schemaProject(),
+		"description":  schemaDescription(),
+		"composite": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Description: "[Composite SLO documentation](https://docs.nobl9.com/yaml-guide/#slo)",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"target": {
+						Type:        schema.TypeFloat,
+						Required:    true,
+						Description: "Designated value",
+					},
+					"burn_rate_condition": {
+						Type:        schema.TypeSet,
+						Optional:    true,
+						Description: "Condition when the Composite SLO’s error budget is burning.",
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								"op": {
+									Type:        schema.TypeString,
+									Required:    true,
+									Description: "Type of logical operation",
+								},
+								"value": {
+									Type:        schema.TypeFloat,
+									Required:    true,
+									Description: "Burn rate value.",
 								},
 							},
 						},
 					},
 				},
 			},
-			"budgeting_method": {
-				Type:        schema.TypeString,
-				Required:    true,
-				Description: "Method which will be use to calculate budget",
-			},
-			"service": {
-				Type:        schema.TypeString,
-				Required:    true,
-				Description: "Name of the service",
-			},
-			"indicator": {
-				Type:        schema.TypeSet,
-				Required:    true,
-				Description: " ",
-				MaxItems:    1,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"name": {
-							Type:        schema.TypeString,
-							Required:    true,
-							Description: "Name of the metric source (agent).",
-						},
-						"project": {
-							Type:        schema.TypeString,
-							Optional:    true,
-							Description: "Name of the metric source project.",
-						},
-						"kind": {
-							Type:        schema.TypeString,
-							Optional:    true,
-							Default:     "Agent",
-							Description: "Kind of the metric source. One of {Agent, Direct}.",
-						},
+		},
+		"budgeting_method": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Method which will be use to calculate budget",
+		},
+		"service": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Name of the service",
+		},
+		"indicator": {
+			Type:        schema.TypeSet,
+			Required:    true,
+			Description: " ",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"name": {
+						Type:        schema.TypeString,
+						Required:    true,
+						Description: "Name of the metric source (agent).",
+					},
+					"project": {
+						Type:        schema.TypeString,
+						Optional:    true,
+						Description: "Name of the metric source project.",
+					},
+					"kind": {
+						Type:        schema.TypeString,
+						Optional:    true,
+						Default:     "Agent",
+						Description: "Kind of the metric source. One of {Agent, Direct}.",
```

Comment:
Is Direct really possible?

Reply:
I guess this was here prior to this change. We should fix all the description issues in a separate PR.